### PR TITLE
fix(php): Anonymous functions without {} block

### LINKED ISSUE #2995: (php) Anonymous functions without {} block not highlighted correctly
**Describe the issue**
Highlighter seems to expect a title in PHP arrow functions, even though they are intentionally anonymous.
Actual: Incorrect title recognized

**Which language seems to have the issue?**
`php`

**Sample Code to Reproduce**
```php
<?php
$fn1 = fn($x) => $x + $y;
```

**Expected behavior**
Expected: Variables

**Additional context**
Found in an [Code-Golf answer](https://codegolf.stackexchange.com/a/218815/25026), but also reproduces in the ungolfed version:
```php
fn( $s ) => 
    ( $l = strlen( $s ) ) > 1 ?             // is input longer than one char?
        max( count_chars( $s ) ) > 1 ?      // has more than 1 of any single char?
            'Bunny'                         // if so, contains dupes so... Bunny it is
        :                                   // else, no dupes
            [
                'T' => [ 'Ka',  'Hu'  ],    // two dimensional associative
                'C' => [ 'Rai', 'Do'  ],    //   array containing handsign 
                'J' => [ 'Hyo',  'Sui' ]    //   and length as indexes
            ]
            [ $s[-1] ]                      // access array by last letter and
            [ $l % 2 ]                      //   length mod 2 (2 == 0, 3 == 1)
            . 'ton'                         //   and append "ton"
    :                                       // finally...
    'Fuma Shuriken'                         // input length was only one char
```

Context: Code Golf source
The comments are colored orange because they are considered `title`.

### PR DESCRIPTION
**Pull request by il3ven**
Resolves #2995

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors

### CODE REVIEW

File: src/languages/php.js [hunk 1]
```diff
@@ -132,11 +132,16 @@ export default function(hljs) {
         className: 'function',
         relevance: 0,
         beginKeywords: 'fn function', end: /[;{]/, excludeEnd: true,
+        keywords: 'fn function',
```
Comment (joshgoebel) [inline src/languages/php.js:135]: This is implied by the previous line and should not be necessary.

File: test/markup/php/comments.txt [hunk 2]
```diff
@@ -17,3 +17,12 @@ function isEven($a) {
 function isOdd($a) {
     return ($a % 2) === 1;
 }
+
```
Comment (joshgoebel) [inline test/markup/php/comments.txt:20]: Please add tests to associated files.  `comments` is meant to test comment highlighting.  It seems we now need a `functions` markup tests added to test function highlighting.  This new test should be placed there - in `functions.txt` and `functions.expect.txt`.

**Review by joshgoebel — CHANGES_REQUESTED** [review]
Looks good other than comments added.

File: src/languages/php.js (hunk 1 above)
Comment (il3ven) [reply src/languages/php.js:135]: Oh, I am so sorry. I forgot to remove this line before doing a commit.

File: test/markup/php/comments.txt (hunk 2 above)
Comment (il3ven) [reply test/markup/php/comments.txt]: Done

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Thanks so much!

**Pull request merged**